en/cpp/quickstart.md

```diff
@@ -85,33 +65,38 @@ git clone https://github.com/mavlink/MAVSDK.git --recursive
 cd MAVSDK
 ```
 
-Build the takeoff and land example:
-```sh
-cd examples/takeoff_land/
-mkdir build && cd build
-cmake ..
-make
+The examples can be found in the examples directory:
+```
+cd examples
 ```
 
+To e.g. build the takeoff and land example, you can do:
+
+```sh
+cd takeoff_land/
+cmake -Bbuild -H.
+cmake --build build -j4
+```
 
-## Running the Example {#running_the_examples}
+## Running an Example {#running_the_examples}
 
 First start PX4 in SITL (Simulation) and *QGroundControl* as described above.
 
 Then run the example app (from the **example/takeoff_land/build** directory) as shown:
 ```sh
-./takeoff_and_land udp://:14540
+build/takeoff_and_land udp://:14540
 ```
 
 The MAVSDK application should connect to PX4, and you will be able to observe the example running in the SDK terminal, SITL terminal, and/or *QGroundControl*.
 The expected behaviour is shown here: [Example: Takeoff and Land](../examples/takeoff_and_land.md).
 
 > **Note** The first argument above is the connection string (`udp://:14540`).
   This is the standard PX4 UDP port for connecting to offboard APIs.
-  
+
```

Comment:
I think we need a specific link or note somewhere here explaining how to write your own apps. I'm thinking a specific section with one paragraph and a link to the en/cpp/guide/toolchain.md doc.

Reply:
For now I've added it in Next Steps just below.